### DISCUSSION TITLE
why i am here?

### DISCUSSION THREAD

**Discussion by clipgit** [community/community#36170] [Other Feature Feedback, Questions, & Ideas]
... i do not know why and what that is in / on here.
now, i can only see and do what others post.
ok, i can not do that so - i can copy it - but thats boring and nonsense.

**Comment by radar-blip** [comment]
You can do whatever you put your mind to. 
Begin.

**Comment by 3kh0** [comment]
This is an open discussion where users can ask for help, share ideas, and more. Give it a try!

**Comment by ghost** [comment]
You are here because you're bored. Pick a project, work on it. Improve your github frontpage. Improve your portfolio...✌️

**Comment by DSDmark** [comment]
Dude... that community is like a drug. First, it's seen as normal, but, then you take it you get addicted
